chore(update): Home Page Promos

dnilasor opened the pull request:
`In this PR, I've updated the conference ad block on the home page to feature information and links for Spinnaker Gardening Days. I've also added a simple Join Slack CTA under that conference ad block, with the slack logo. Ideally this block would be a bit narrower so that it doesn't push other page content down so much, but to me this is acceptable in the short-term while we refresh the site. My reasoning for adding the CTA is that, I'm worried folks are coming to the site home page now and not knowing where to turn for help. We can help OSS users much better on Slack, so they don't get stuck and frustrated on their own.
@brian-armory @dorbin @aimeeu take a peek if you like.

dnilasor committed:
update conf ad row for Spinnaker Gardening

dnilasor committed:
add hackathon image

dnilasor committed:
add slack join row

dnilasor committed:
add slack logo

dnilasor committed:
add include

dnilasor committed:
syntax tweaks

317brian committed:
copyedit

317brian commented:
LGTM. One question before I approve and merge, is the image public domain or some other free use license?

317brian committed:
Merge branch 'master' into dnilasor-home-promos

dorbin commented on index.md:
```diff
@@ -1,10 +1,15 @@
 ---
 layout: home
 conference_ad_row:
```
@dnilasor do you mean to keep the previous summit-row name? No big deal if you don't change it.

dnilasor replied:
I did because it seemed like a generic-enough row name. Maybe I should change it to an event more generic name (promo_row ?) so we don't need to change it? @dorbin maybe there is some value there

dorbin reviewed (approved):
LGTM, with one small comment. 
Thanks!

dnilasor committed:
Adding trackable slack join link

@brian-armory @dorbin I received a request that we be able to track how many folks join slack using the link from spinnaker.io. The easiest way for me to do this currently is to use Armory's existing rebrandly account which includes detailed link tracking, so I've added that URL here. I'm sure there is a way to do this with Google Analytics and Tag Manager, although I don't know it off the top of my head. I want to know, are we comfortable with me using the rebrandly approach if we're not using it for anything other than tracking? Or do I need to look into an alternative approach? Thanks!

dnilasor committed:
add generic language for promo row

I don't see any reason to change this every time if there are generic labels that make sense?

dnilasor commented:
I added another change with a descriptive message. Please LMK if anything else needed before approval!

dnilasor committed:
Merge branch 'master' into dnilasor-home-promos

dorbin commented on index.md:
```diff
@@ -1,12 +1,17 @@
+  alt: "Spinnaker Gardening Days Logo"
+slack_row:
+  title: Join Spinnaker Slack
+  excerpt: "Visit [join.spinnaker.io](https://go.armory.io/spinslack) to get help on Spinnaker Slack, a place for the community to come together. Use this vibrant workspace to ask and answer questions, connect with other operators and users, discuss issues with SIGs, and learn Spinnaker!"
```
Hm, might be better to link directly to join.spinnaker.io rather than to an Armory url that redirects there?

dnilasor replied:
@dorbin understood, here's my commit message on this:

>  I received a request that we be able to track how many folks join slack using the link from spinnaker.io. The easiest way for me to do this currently is to use Armory's existing rebrandly account which includes detailed link tracking, so I've added that URL here. I'm sure there is a way to do this with Google Analytics and Tag Manager, although I don't know it off the top of my head. I want to know, are we comfortable with me using the rebrandly approach if we're not using it for anything other than tracking? Or do I need to look into an alternative approach? Thanks!

I'd love your thoughts @dorbin @brian-armory !

dorbin commented on index.md:
```diff
@@ -1,12 +1,17 @@
+  alt: "Join Spinnaker Slack"
 spinnaker_row:
-  title: Spinnaker is an open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
+  title: Spinnaker is an open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and nczidence.
```
Inadvertent?

dnilasor replied:
Yes, wth oops

dnilasor replied:
fixed

dorbin commented on index.md:
```diff
@@ -1,12 +1,17 @@
-  excerpt: "Spinnaker Summit 2019 took place in San Diego, CA from November 15-17th. Hundreds of people from the Spinnaker community gathered and shared stories about their experiences with Spinnaker and Continuous Delivery. See the insights from Spinnaker contributors and users in [full session recordings here](https://www.youtube.com/playlist?list=PL4yLrwUObNkvGzXKSEU8miQTlvZyZvXWq)."
-  image_path: assets/images/summit-socialmedia-transparent.png
-  alt: "Spinnaker Summit Logo"
+promo_row:
```
Yeah, this!

dorbin commented on index.md:
```diff
@@ -1,12 +1,17 @@
+  alt: "Spinnaker Gardening Days Logo"
+slack_row:
+  title: Join Spinnaker Slack
+  excerpt: "Visit [join.spinnaker.io](https://go.armory.io/spinslack) to get help on Spinnaker Slack, a place for the community to come together. Use this vibrant workspace to ask and answer questions, connect with other operators and users, discuss issues with SIGs, and learn Spinnaker!"
```
```suggestion
  excerpt: "Visit [join.spinnaker.io](https://go.armory.io/spinslack) to get help on Spinnaker Slack, a place for the community to come together. Use this vibrant workspace to ask and answer questions, connect with other operators and users, discuss issues with SIGs, and learn about Spinnaker!"
```

dorbin replied:
Just because "..and learn Spinnaker" almost implies Spinnaker Slack is sufficient for learning Spinnaker.

dnilasor committed:
fix typo introduced in this pr

dnilasor committed:
syntax specificity

Co-Authored-By: Dave Dorbin <ddorbin@google.com>

dnilasor commented:
@plumpy, makes sense. Thanks for putting #1646 on my radar. Seems like we need to sort out implementation details, so for now I'll just re-add the simple join.spinnaker.io link so we can get this approved. Then I'll look for the most maintainable strategy. Thanks for looking @dorbin @plumpy !!

dnilasor committed:
use simple join link for now

dnilasor commented:
@brian-armory, changed the link to something not controversial and will open another PR that addresses this and #1646. Mergey merge time please?

dorbin committed:
Merge branch 'master' into dnilasor-home-promos

317brian commented:
Thanks for making sure the analytics discussion doesn't get lost @dnilasor. I'll go ahead and merge since Dave already approved.

317brian commented:
Weird. What appears on the OSS site is not what showed locally when I built it originally. @dnilasor will troubleshoot.
